python/nano/test/tf/keras/test_trace_and_quantize.py

```diff
@@ -99,3 +108,16 @@ def test_evaluate_after_trace(self):
             InferenceOptimizer.save(traced_model, tmp_dir_name)
             new_model = InferenceOptimizer.load(tmp_dir_name, model)
         new_model.evaluate(x=x, y=y)
+
+    def test_inference_output_shape(self):
+        model = MyModelReturnList()
+        x = np.random.random((100, 4))
+        traced_model = InferenceOptimizer.trace(model, accelerator="onnxruntime",
+                                                input_spec=tf.TensorSpec(shape=(None, 4), dtype=tf.float32))
+        outputs = traced_model(x, tf.convert_to_tensor(True))
```

Comment:
why here needs an additional `tf.convert_to_tensor(True)`?

Reply:
A typo, fixed it